lib/super_digit.rb

```diff
@@ -1,9 +1,11 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
This works but is O(d^2) where d is the number of digits because you have a sum O(d) inside a recursive method which goes d times.